### PR TITLE
Issue #19 - Add Speakers List to website

### PR DESCRIPTION
**Pull request by daminisatya**
The images which are unavailable in any of their accounts is displayed with a default image. 
The speakers.json is updated from here @joerx 
https://github.com/fossasia/open-event-scraper/blob/master/out/speakers.json

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1259,6 +1267,54 @@ <h1 class="text-white">FOSSASIA is the annual OpenTechSummit featuring prominent
...
+	        }
+	            $('#speakersList').append(speakerString);
+	        } 
+	    });
+		function imgError(image) {
+ 		    image.onerror = "";
+		    image.src = "https://placeholdit.imgix.net/~text?txtsize=28&txt=300%C3%97300&w=300&h=300";;
+		    return true;
+        }
+        </script>
```
Comment (sudheesh001) [inline index.html:1317]: indentation of this function is off compared to the `$.ajax()` query

Comment (daminisatya) [inline index.html:1317]: @sudheesh001 changed !

**Comment by sudheesh001** [comment]
@joerx This seems to work, but quite a few images break and are replaced with the default onerror image but I think that issues can be resolved quite easily by just renaming the files accordingly in the `img/` folder so that we could get this up and running.

**Comment by sudheesh001** [comment]
You could also delete the following lines @daminisatya 
https://github.com/daminisatya/2016.fossasia.org/blob/updateSpeakersList/index.html#L544-L692

File: index.html [hunk 2]
```diff
@@ -1259,6 +1267,54 @@ <h1 class="text-white">FOSSASIA is the annual OpenTechSummit featuring prominent
         <script src="js/tweetsfetcher.js"></script>
         <script src="js/placeholders.min.js"></script>
         <script src="js/scripts.js"></script>
+        <script>
+        $.ajax({
+	        type : 'GET',
+	        dataType : 'json',
+	        url: 'schedule/speakers.json',
```
Comment (joerx) [inline index.html:1274]: Please fetch the speaker list from https://raw.githubusercontent.com/fossasia/open-event-scraper/master/out/speakers.json

Comment (daminisatya) [inline index.html:1274]: @joerx Ok will do it.

File: index.html [hunk 3]
```diff
@@ -1259,6 +1267,54 @@ <h1 class="text-white">FOSSASIA is the annual OpenTechSummit featuring prominent
...
+	            		var found = blockNameList.indexOf(names[nameValue].name);
+	            		console.log(found + ' ' + names[nameValue].name);
+	            		if(found === -1) {
+		            		speakerString += "<div class=\"col-md-3 col-sm-6 speaker-column\">"+
+										 		"<div class=\"speaker\">"+
+									    			"<div class=\"image-holder\">"+
+									    				"<img class=\"background-image\" height=\"300\" width=\"300\" alt=\""+x+"\" src=\"img\/"+x+"\" onerror=\"imgError(this)\">"+
+									     				"<div class=\"hover-state text-center preserve3d\">"+
+															"<div class=\"social-links vertical-align\">"+
+																"<a href=\""+names[nameValue].twitter+"\"><i class=\"icon social_twitter\"></i></a>"+
```
Comment (aviaryan) [inline index.html:1297]: Don't show twitter icon for those who haven't given a twitter profile. Same goes for LinkedIn.

**Comment by joerx** [comment]
We should stick to the static speaker list. Hand-curating the images leads to better results and we don't need to have every single speaker on the page. I suggest to close this PR

**Comment by sudheesh001** [comment]
@joerx Since it's a static speakers list, I agree with you, this is best to close :smile:

**Pull request closed without merging**